CHANGELOG.next.toml

```diff
@@ -29,6 +29,42 @@ references = ["aws-sdk-rust#317", "smithy-rs#907"]
 meta = { "breaking" = false, "tada" = false, "bug" = true }
 author = "jdisanti"
 
+[[aws-sdk-rust]]
+message = '''
+If you directly depend on AWS or Smithy runtime crates _(e.g., AWS crates not named `aws-config` or prefixed with `aws-sdk-`)_,
+the formerly default features from those crates must now be explicitly set in your `Cargo.toml`.
+
+
+**Upgrade guide**
+
+| before                          | after |
+|---------------------------------|-------|
+| `aws-sigv4 = "VERSION"`         | `aws-sigv4 = { version = "VERSION", features = ["sign-http"] }` |
+| `aws-smithy-async = "VERSION"`  | `aws-smithy-async = { version = "VERSION", features = ["rt-tokio"] }` |
+| `aws-smithy-client = "VERSION"` | `aws-smithy-client = { version = "VERSION", features = ["client-hyper", "rustls", "rt-tokio"] }` |
+| `aws-smithy-http = "VERSION"`   | `aws-smithy-http = { version = "VERSION", features = ["rt-tokio"] }` |
+'''
+references = ["smithy-rs#930"]
+meta = { "breaking" = true, "tada" = false, "bug" = false }
+author = "zhessler"
+
+[[smithy-rs]]
+message = '''
+Runtime crates no longer have default features. You must now specify the features that you want when you add a dependency to your `Cargo.toml`.
+
+**Upgrade guide**
+
+| before                          | after |
+|---------------------------------|-------|
+| `aws-sigv4 = "VERSION"`         | `aws-sigv4 = { version = "VERSION", features = ["sign-http"] }` |
```

Comment:
`aws-sigv4` is only for the SDK, so it can be removed from the smithy-rs section.
```suggestion
```